workflows/spaceranger/spaceranger_count.wdl

```diff
@@ -232,8 +232,15 @@ task run_spaceranger_count {
             return darkimages
 
         has_cyta = not_null('~{cytaimage}')
+        probe_set = "~{probe_set}"
+        probe_set = "" if probe_set is None else probe_set
```

Comment:
No need to check if `probe_set` is `None`. You can drop this line.

Reply:
It fails without it if s none, hence why is there

Reply:
to me is also counter-intuitive, but that is what I got